package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "scrambling-letters",
   "version": "1.6.1",
+  "main": "dist/scrambler.cjs.js",
+  "module": "dist/scrambler.ems.js",
```

Comment:
Thanks for the PR brianbento :) I think this should be ESM not EMS though?

Reply:
My bad.